### DISCUSSION TITLE
수정한 소스가 크롬 주소창에서 실행이 안됩니다.

### DISCUSSION THREAD

**Discussion by hky0102** [kimytsc/covid-rest-vaccine-macro#54] [Q&A]
아래와 같이 수정한 소스를 크롬 주소창에 붙여 넣었는데 실행이 되지 않습니다.(javascript 맨앞에 붙여 주었습니다.)
뭐가 잘못된걸까요 ? 도와주세요.

javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=unused&x=126.981642&y=37.560182&query=%EC%9E%94%EC%97%AC%EB%B0%B1%EC%8B%A0&sessionId=x5kha2tY1bDPZXTk9Mv8yg%3D%3D&bounds=126.9794308%3B37.557724%3B126.9838532%3B37.5626399",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",/화이자/
        "VEN00014",/모더나/
]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/naver/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()

**Comment by kimytsc** [comment]
잘못되었다면 2가지 중에 하나로 보입니다.

첫번째로, `예약 신청 페이지`에서 진행하지 않았을 가능성이 있으니, README의 `예약 신청 페이지`를 참고하여 접근하신 후 실행시켜주세요.

두번째로, choice; [
"VEN00013",/화이자/
"VEN00014",/모더나/
] 부분을 `choice: ["VEN00013","VEN00014"]` 로 바꿔보시겠어요?

원 소스는 `/*화이자*/`로 되어 있었는데 중간에 복붙 하는 과정에서 `*`이 빠지고 `/화이자/`로 바뀌어서 그런 것 같습니다.
제가 변경해서 테스트해보니 잘 되는것을 확인했습니다.
